shared/folders/files/file/render.desktop.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import React, {Component} from 'react'
+import {Box, Text, Icon, ListItem} from '../../../common-adapters'
+import {globalStyles, globalColors} from '../../../styles/style-guide'
+import type {Props} from './render'
+
+export default class Render extends Component<void, Props, void> {
+  _renderIconPart () {
+    const size = this.props.size === 'Small' ? 32 : 48
+    return <Icon type={this.props.fileIcon} style={{height: size, width: size}} />
+  }
+
+  _renderBody () {
+    return (
+      <Box>
+        <Text type='BodySmallSemibold' style={filenameStyleThemed[this.props.theme]}>{this.props.name}</Text>
```

Comment:
when i did this in the main folder list i pulled this out into a style variable in the render() function and passed it to the sub renderers. so you only need the dereference once, so like
```const styles = style[this.props.theme]```
then
```this._renderBody(styles)```